static/js/partners.js

```diff
@@ -93,8 +93,8 @@ partners.loadJSONP = function(uri, callback) {
 
 
 // Check partnerLogoClouds is available and if not produce warning
-if (typeof(partnerLogoClouds) === 'object') {
-    partners.loadPartnerClouds();
-} else {
-    console.warn('Load partner logos: Expected array "partnerLogoClouds" not found. Stopping.');
-}
+// if (typeof(partnerLogoClouds) === 'object') {
```

Comment:
Likewise, it would be nice not to keep commented out code, but I'm happy to tidy this up myself afterwards, since I need to do some work for the managed cloud calculator anyway,